src/go/rpk/pkg/cli/cmd/wasm/deploy_test.go

```diff
@@ -0,0 +1,214 @@
+package wasm
+
+import (
+	"bytes"
+	"fmt"
+	"testing"
+
+	"github.com/Shopify/sarama"
+	"github.com/Shopify/sarama/mocks"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/afero"
+	"github.com/stretchr/testify/require"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/kafka"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/utils"
+)
+
+type fileInfo struct {
+	name	string
+	content	string
+}
+
+type mockAdmin struct {
```

Comment:
I thought you would put this in `kafka/test.go` like I showed you with `container/common/test.go`

Reply:
Got it, I'll do it

Reply:
I created `kafka/mocks/admin`